lib/atmosphere/engine.rb

```diff
@@ -39,6 +39,14 @@ class Engine < ::Rails::Engine
       g.helper false
     end
 
+    initializer :append_migrations do |app|
+      unless app.root.to_s.match root.to_s
+        config.paths["db/migrate"].expanded.each do |expanded_path|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.